modules/promotions/assets/js/react/components/promotion-card.js

```diff
@@ -0,0 +1,66 @@
+import {
+	Card,
+	CardHeader,
+	Chip,
+	CardMedia,
+	CardContent,
+	Typography,
+	CardActions,
+	Button,
+	Box,
+	CloseButton,
+} from '@elementor/ui';
+
+const PromotionCard = ( { onClose, promotionsData } ) => {
+	const title = promotionsData?.title;
+	const description = promotionsData?.description;
+	const imgSrc = promotionsData?.image;
+	const ctaText = promotionsData?.upgrade_text;
+	const ctaUrl = promotionsData?.upgrade_url;
+
+	const redirectHandler = () => {
+		return window.open( ctaUrl, '_blank' );
+	};
+
+	return (
+		<Card sx={ { width: 296 } }>
+			<CardHeader title={
+				<Box sx={ { display: 'flex', alignItems: 'center', justifyContent: 'space-between' } }>
```

Comment:
Maybe;

``
```suggestion
				<Box display="flex" alignItems="center" justifyContent="space-between">
```

See e.g. https://github.com/elementor/elementor-pro/blob/fb6d487c86e83ac9506ae3845f83aa663794568f/modules/display-conditions/assets/js/editor/components/conditions.js#L10